### ISSUE TITLE
If I set transition-delay: 0

### ISSUE THREAD

**Issue by Geekhyt** [stylelint/stylelint#4898]
> Clearly describe the bug

If I set transition-delay:0, I expect the editor to have an error message, because if I set transition-delay to 0, it is wrong.Because the default value of transition-delay is 0s, if it is set to 0, this statement will not take effect in the browser.

> Which rule, if any, is the bug related to?

I don't know much

> What code is needed to reproduce the bug?

```css
transition: background-color 0.6s ease-in-out 0;
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
    "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

+ stylelint@13.6.1
+ stylelint-config-standard@20.0.0

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

npx stylelint ...

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

I hope there will be an error message.

> What actually happened (e.g. what warnings or errors did you get)?

Actually there is no error message

**Comment by jeddy3** [comment]
@Geekhyt Thanks for the report and for using the template.

You can use the [stylelint-csstree-validator plugin](https://www.npmjs.com/package/stylelint-csstree-validator) to do this. The rules built-in to stylelint [will only detect the simpliest validation errors](https://stylelint.io/about/linting#validators).

**Comment by Geekhyt** [comment]
> @Geekhyt Thanks for the report and for using the template.
> 
> You can use the [stylelint-csstree-validator plugin](https://www.npmjs.com/package/stylelint-csstree-validator) to do this. The rules built-in to stylelint [will only detect the simpliest validation errors](https://stylelint.io/about/linting#validators).

Thank you very much！
